client/scss/components/_image_viewer.scss

```diff
@@ -6,25 +6,39 @@
   background-color: color('black');
   width: 100vw;
   height: 100vh;
-  display: none;
   position: fixed;
+  display: none;
   z-index: 1000;
   top: 0;
   left: 0;
+  transition: transform 500ms;
+}
+
+.image-viewer__content2 {
+  display:block;
 }
 
 .image-viewer__controls {
   height: 3em;
-
+  
```

Comment:
I think you can set up your editor to trim whitespace on otherwise empty lines

Reply:
👍